--- a/src/luaconf.h
+++ b/src/luaconf.h
@@ -35,12 +35,12 @@
 #ifndef LUA_LMULTILIB
 #define LUA_LMULTILIB	"lib"
 #endif
-#define LUA_LROOT	"/usr/local"
-#define LUA_LUADIR	"/lua/5.1/"
+#define LUA_LROOT	"/usr"
+#define LUA_LUADIR	"/lua/"
 #define LUA_LJDIR	"/luajit-2.1.0-beta3/"
 
 #ifdef LUA_ROOT
 #define LUA_JROOT	LUA_ROOT
-#define LUA_RLDIR	LUA_ROOT "/share" LUA_LUADIR
+#define LUA_RLDIR	LUA_ROOT "/lib" LUA_LUADIR
 #define LUA_RCDIR	LUA_ROOT "/" LUA_MULTILIB LUA_LUADIR
 #define LUA_RLPATH	";" LUA_RLDIR "?.lua;" LUA_RLDIR "?/init.lua"
